torch/_meta_registrations.py

```diff
@@ -105,6 +105,20 @@ def meta_index_select_out(self, dim, index, out):
     return out.copy_(torch.index_select(self, dim, index))
 
 
+@register_meta(
+    [
+        aten.relu_.default,
+        aten.add_.Tensor,
+        aten.hardtanh_.default,
+        aten.hardswish_.default,
+        aten.leaky_relu_.default,
+    ],
+    register_dispatcher=False,
+)
+def meta_inplace_(self, *args, **kwargs):
+    return self
```

Comment:
This is incomplete, not enough error checking

Reply:
uh... I guess this is fine for unary ops? Or I suppose unary ops need to check dtypes as well. Annoying.

Reply:
I guess we should just make full decompositions for some of these lol.

Reply:
Yeah, dtypes. Ofc I know our error checking is woefully inadequate, will have to be addressed differently...